spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/metrics/MetricsEndpoint.java

```diff
@@ -80,8 +80,12 @@ private String getName(Meter meter) {
 	@ReadOperation
 	public MetricResponse metric(@Selector String requiredMetricName,
 			@Nullable List<String> tag) {
-		Assert.isTrue(tag == null || tag.stream().allMatch((t) -> t.contains(":")),
-				"Each tag parameter must be in the form key:value");
+		if (tag != null && tag.stream().anyMatch((t) -> !t.contains(":"))) {
```

Comment:
Perhaps we should move that to the `parseTag` method and add a test for it? the existing code that checks there is `:` in the String is a bit weak as well.

Reply:
@snicoll Thanks for the quick feedback! I updated as you suggested.